src/main/java/com/google/starfish/services/TableService.java

```diff
@@ -97,4 +134,38 @@ public boolean deleteRowById(DataSource pool, long id) {
       return false;
     }
   }
+
+  /** Deletes row by string id from sql */
+  public boolean deleteRowById(DataSource pool, String id) {
+    try (Connection conn = pool.getConnection()) {
+      try {
+        conn.setAutoCommit(false);
+        String stmt = 
+            "DELETE * "
+          + "FROM starfish." + this.sqlTable.getSqlTable() + " "
```

Comment:
nit: I don't think we need the `starfish.` in the query

Reply:
You're right, good catch!